generate.go

```diff
@@ -646,8 +648,53 @@ func getDefaultTemplate() specs.Spec {
 			},
 		},
 		Hostname: "shell",
-		Mounts:   []specs.Mount{},
+		Mounts: []specs.Mount{
+			{
+				Destination: "/proc",
+				Type:        "proc",
+				Source:      "proc",
+				Options:     nil,
```

Comment:
It looks like this is rolling back 080ee00 (drop-runtime-supplied-devices-and-mounts, #2).  Did we miss something there?